exploration-module/src/main/java/io/edpn/backend/exploration/application/domain/LandingPadSize.java

```diff
@@ -0,0 +1,17 @@
+package io.edpn.backend.exploration.application.domain;
+
+public enum LandingPadSize {
+    UNKNOWN,
+    SMALL,
+    MEDIUM,
+    LARGE;
+
+    public static LandingPadSize fromString(String landingPadSize) {
+        return switch (landingPadSize.toUpperCase()) {
```

Comment:
I guess `LandingPadSize.name()` can't be leveraged in the case labels. While the `switch` looks nicer, the literals could be avoided:
```
String upper = landingPadSize.toUpperCase();
Stream.of(LandingPadSize.values())
    .filter(size -> size.name().equals(upper))
    .findAny().orElse(UNKNOWN);
```

Perhaps could also provide a constant for the "known values", so that filtering out `UNKNOWN` can be avoided?